client/src/App.js

```diff
@@ -1,24 +1,25 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
+import React, { Component } from "react";
+import { connect } from "react-redux";
 import { Route, NavLink } from "react-router-dom";
-import { withRouter } from 'react-router';
-import Home from './components/Home';
-import Nav from './components/nav';
-import Users from './components/users/Users';
-import Register from './components/users/Register';
-import Post from './components/posts/Post';
-import CreatePost from './components/posts/CreatePost.js';
-import { Button } from 'reactstrap';
+import { withRouter } from "react-router";
+import Home from "./components/Home";
+import Nav from "./components/nav";
+import Users from "./components/users/Users";
+import Register from "./components/users/Register";
+import Post from "./components/posts/Post";
+import CreatePost from "./components/posts/CreatePost.js";
+import { Button } from "reactstrap";
 
-import { getPosts, getUsers, getTest, getPost } from './actions';
-import SearchResults from './components/posts/SearchResults';
+import { getPosts, getUsers, getTest, getPost } from "./actions";
+import SearchResults from "./components/posts/SearchResults";
 
+import { getPosts, getUsers, getTest, getPost, login } from "./actions";
```

Comment:
Build preview is failing because these imports are duplicated on line 13 and 16.